src/templates/_common/includes/tracker.jsx

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable */
+import React from 'react';
+
+const Tracker = () => (
```

Comment:
maybe call it `GoogleOptimizer` and comment what it does: https://support.google.com/optimize/answer/7100284

Reply:
yes, thats sounds better